app.R

```diff
@@ -379,7 +379,13 @@ schema_to_display_lookup <- data.frame(schema_name, display_name)
       filename_list <- names(file_namedList)
 
 
-      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), template_type, filenames = as.list(filename_list))
+      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), 
+						      template_type, 
+						      filenames = as.list(filename_list),
+						      datasetId = folder_synID,
+						      filenames = as.list(filename_list),
+						      useAnnotations = F,
+						      sheetUrl = T)
```

Comment:
Per the new `getModelManifest()` [signature](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/models/metadata.py#L115), we can remove explicit specification of both `useAnnotations` and `sheetUrl`.